pages/contacts/[id]/edit/index.js

```diff
@@ -86,17 +85,15 @@ export default function EditPage() {
 
   return (
     <>
-      <StyledAppBodyContainer>
-        <ContactForm
-          onSubmit={handleEdit}
-          formName={"edit-contact"}
-          defaultData={contact}
-        />
-        <StyledButtonsContainer>
-          <StyledButton onClick={submitDelete}>loschen</StyledButton>
-        </StyledButtonsContainer>
-        <Navigation />
-      </StyledAppBodyContainer>
+      <ContactForm
+        onSubmit={handleEdit}
+        formName={"edit-contact"}
```

Comment:
if "edit-contact" is a string you can get rid of the {}

Reply:
you're right, the habit of putting them everywhere.🤣